app/slashcommands/command_invite_people.go

```diff
@@ -23,11 +23,11 @@ func init() {
 	app.RegisterCommandProvider(&InvitePeopleProvider{})
 }
 
-func (me *InvitePeopleProvider) GetTrigger() string {
+func (p *InvitePeopleProvider) GetTrigger() string {
```

Comment:
Oh I see, it's just an empty struct. In that case, I would suggest to just remove the receiver names altogether. There's no use keeping them if we aren't using them.

Reply:
I have updated the commit, and have **removed the receiver names**